src/lib/stitching/vortex/stitching.ts

```diff
@@ -297,13 +300,15 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
           const typename = parent.rankedEntity.__typename
           const fieldName = removeVortexPrefix(typename).toLowerCase()
           const id = parent.rankedEntity.entityId
+          const partnerId = parent.rankedEntity.partnerId
```

Comment:
I was thinking if it is possible to get that `partnerId` from 2 levels up `Parent` type since it is already [available there](https://github.com/artsy/metaphysics/blob/1ea52c5af351154337d900367e45e6e097c5d09c/src/lib/stitching/vortex/stitching.ts#L239-L241) in the query and there is no need to get it back again from vortex. (sample query:
```graphql
query {
  partner(id: "4d8b92c44eb68a1b2c0004cb") {
    analytics {
      topShows: rankedStats(first: 1, objectType: SHOW, period: ONE_YEAR) {
        edges {
          node{
            entity {
              ... on Show{
                name
                images {
                  image_url
                }
              }
            }
          }
        }
      }

    }
  }
}
```

Just searching a little bit I found [similar examples](https://github.com/graphql/graphql-js/issues/1098) but that is not on a merged schema so I don't know in this case that `parent` is filled in by the schema merging (I _think_) is it possible to do that? cc: @alloy @ashkan18